src/code/compareSimulations.R

```diff
@@ -0,0 +1,498 @@
+#objectType can be "Parameter", "Species", or "Observer"
+compareStructures = function(objectType, DCI_Info1, DCI_Info2, simName1, simName2, numTolerance, ignoreFormula){
+  #This list holds information about the differences between simulations objects.
+  objects_diff = list();
+  
+  #Get existence information about all objects depending on their type.
+  if(objectType == "Parameter"){
+    allObjects1 = existsParameter(path_id = "*", options = list(Type = "readonly"), DCI_Info = DCI_Info1);
+    allObjects2 = existsParameter(path_id = "*", options = list(Type = "readonly"), DCI_Info = DCI_Info2);
+  }
+  if(objectType == "Species"){
+    allObjects1 = existsSpeciesInitialValue(path_id = "*", options = list(Type = "readonly"), DCI_Info = DCI_Info1);
+    allObjects2 = existsSpeciesInitialValue(path_id = "*", options = list(Type = "readonly"), DCI_Info = DCI_Info2);
+  }
+  if(objectType == "Observer"){
+    allObjects1 = existsObserver(path_id = "*", DCI_Info = DCI_Info1);
+    allObjects2 = existsObserver(path_id = "*", DCI_Info = DCI_Info2);
+  }
+  
```

Comment:
Exception should be thrown if objectType  is not in {"Parameter", "Species", "Observer"}

Reply:
Added.